debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -163,6 +165,15 @@ else if (message.getOperation() == Operation.COMMIT) {
                         offsetContext.updateWalPosition(lsn, lastCompletelyProcessedLsn, message.getCommitTime(), message.getTransactionId(), tableId,
                                 taskContext.getSlotXmin(connection));
 
+                        Set<Envelope.Operation> skippedOperations = taskContext.getConfig().getSkippedOps();
+                        if (skippedOperations != null) {
+                            for (Envelope.Operation operation : skippedOperations) {
+                                if (operation == Envelope.Operation.CREATE || operation == Envelope.Operation.UPDATE || operation == Envelope.Operation.DELETE) {
```

Comment:
Hum, this doesn't really make sense; we'd skip *any* record as soon as there's a single skipped op configured. Also, this should go into `EventDispatcher`, which allows to implement this feature in a single place (MySQL was a bit special, as it isn't using the framework structure around the dispatcher yet).

Reply:
I couldn't find `EventDispatcher` where is it?

Reply:
In the core module, see `io.debezium.pipeline.EventDispatcher`.